components/kyma-environment-broker/internal/storage/storage.go

```diff
@@ -15,6 +15,7 @@ type BrokerStorage interface {
 	Provisioning() Provisioning
 	Deprovisioning() Deprovisioning
 	LMSTenants() LMSTenants
+	Orchestration() Orchestration
```

Comment:
maybe let's keep the plural form here as we did with other resources

```suggestion
	Orchestrations() Orchestrations
```